### PR TITLE
#165273530: social login and register

### PR DESCRIPTION
**Pull request by JamesMudidi**
**What does this PR do?**

- Implements user signup and sign in using social authentication

**Description of Task to be completed?**

- [x] create login buttons for facebook and google
- [x] create login actions for facebook and google
- [x] write tests for login buttons
- [x] write tests login actions

**How should this be manually tested?**

- Go to the review app [here](https://ah-frontend-dojo-pr-18.herokuapp.com)
- Navigate to the login route and click on any social button

**Any background context you want to provide?**

- This is currently a work in progress. Many things are still being fixed and refactored 

**What are the relevant pivotal tracker stories?**

- [#165273530](https://www.pivotaltracker.com/story/show/165273530)

**Screenshots (if appropriate)**
Screenshot 2019-06-14 at 7 42 29 AM

### CODE REVIEW

File: src/components/Login/index.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ import 'components/Login/Login.scss';
 
 // Modal Component
 import ModalBox from 'components/ModalBox';
+import FaceBookButton from '../Social/Facebook/Facebook';
+import GoogleButtons from '../Social/Google/Google';
```
Comment (kalsmic) [inline src/components/Login/index.js]: @JamesMudidi, please consider revising the imports to
```
import FaceBookButton from 'components/Social/Facebook/Facebook';
import GoogleButtons from 'components/Social/Google/Google';
```

File: src/components/Social/Google/GoogleStyle.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.googlebutton > button {
+  box-shadow: 0px 0px 0px 0px !important;
+  background-color: #EBEBEB !important;
+
+  > div {
+    background-color: #EBEBEB !important;
+  }
+}
```
Comment (kalsmic) [inline src/components/Social/Google/GoogleStyle.scss]: Please also kindly consider adding a line at the end of the file.

**Review by kalsmic — CHANGES_REQUESTED** [review]
Well done, @JamesMudidi, please kindly consider looking into these issues.

**Review by llwasampijja — CHANGES_REQUESTED** [review]
Good work @JamesMudidi on the implementation of the feature as far. However, it would be nice if you documented your code. Otherwise kudos!

**Pull request closed without merging**